dpctl/tests/test_usm_ndarray_ctor.py

```diff
@@ -1402,7 +1402,7 @@ def test_triu_order_k(order, k):
     Xnp = np.arange(np.prod(shape), dtype="int").reshape(shape, order=order)
     Ynp = np.triu(Xnp, k)
     assert Y.dtype == Ynp.dtype
-    assert X.flags == Y.flags
+    assert X.flags.flags == Y.flags.flags
```

Comment:
Perhaps this change suggest that `_flags` object can use `__equal__` method implementation.